CONTRIBUTING.md

```diff
@@ -181,13 +181,23 @@ follow the instructions in this section.
 
 ### Update the version, changelog, and notices
 1. Create a new branch for the version bump.
-1. Based on the unreleased content, determine the new version number and update
-   the [version](pkg/secrets/version.go) file.
+1. Based on the unreleased content, determine the new version number.
+1. Update this version in the following files:
+    1. [Version file](pkg/secrets/version.go)
+    1. [Chart version](helm/secrets-provider/Chart.yaml)
+    1. [Default deployed version](helm/secrets-provider/values.yaml)
+    
+1. Create a helm package by running the following command from the repo root: `pushd helm/secrets-provider/packages && helm package .. && popd`
```

Comment:
@cyberark/palmtree I encourage you to add this step to the pipeline and use `archiveArtifacts` to save the package - that ensures that the dev's local environment doesn't pollute the artifact and it's consistently built on a clean executor

I'm excited to have another helm chart to share!!

Reply:
@eladkug FYI

Reply:
@izgeri can you send me an example of where we do this? Was looking here https://github.com/cyberark/conjur-oss-helm-chart/blob/master/Jenkinsfile to find a reference

Reply:
we do it in secretless here: https://github.com/cyberark/secretless-broker/blob/master/Jenkinsfile#L196

we're just beginning to propagate this, but it's part of our effort to automate releases as much as possible

in the helm chart it's actually running as a github action: https://github.com/cyberark/conjur-oss-helm-chart/blob/master/.github/workflows/package.yml

Reply:
To follow the secretless example from `https://github.com/cyberark/secretless-broker/blob/master/Jenkinsfile#L196`, our `bin/build_release` would just have that helm package command? Once we have this package though we would need to push https://github.com/cyberark/helm-charts. Could this also be automated?